File: app/src/main/java/com/example/walpaper_kotlin/service/model/Sponsorship.java
package com.example.walpaper_kotlin.service.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Sponsorship {

    @SerializedName("impression_urls")
    @Expose
    private List<String> impressionUrls = new ArrayList<String>();
    @SerializedName("tagline")
    @Expose
    private String tagline;
    @SerializedName("sponsor")
    @Expose
    private Sponsor sponsor;

    public List<String> getImpressionUrls() {
        return impressionUrls;
    }

    public void setImpressionUrls(List<String> impressionUrls) {
        this.impressionUrls = impressionUrls;
    }

    public String getTagline() {
        return tagline;
    }

    public void setTagline(String tagline) {
        this.tagline = tagline;
    }

    public Sponsor getSponsor() {
        return sponsor;
    }

    public void setSponsor(Sponsor sponsor) {
        this.sponsor = sponsor;
    }

}

File: app/src/main/java/com/example/service/model/Sponsorship.kt
package com.example.service.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

class Sponsorship {
    @SerializedName("impression_urls")
    @Expose
    var impressionUrls: List<String> =
        ArrayList()

    @SerializedName("tagline")
    @Expose
    var tagline: String? = null

    @SerializedName("sponsor")
    @Expose
    var sponsor: Sponsor? = null

}